.pylintrc

```diff
@@ -9,7 +9,7 @@
 
 # Add files or directories to the blacklist. They should be base names, not
 # paths.
-ignore=.git,sphinxtogithub
+ignore=.git,qtpy,sphinxtogithub
```

Comment:
Hello, you are targetting a random `simplify-pynsist-preamble` branch which has already been merged into `main`.  Please rebase against `main` and resubmit.